pkg/kube/apisix/apis/config/v2beta1/types.go

```diff
@@ -197,3 +197,46 @@ type ApisixRouteList struct {
 	metav1.ListMeta `json:"metadata" yaml:"metadata"`
 	Items           []ApisixRoute `json:"items,omitempty" yaml:"items,omitempty"`
 }
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:subresource:status
+
+// ApisixPluginConfig is the Schema for the ApisixPluginConfig resource.
+// An ApisixPluginConfig is used to support a group of plugin configs
+type ApisixPluginConfig struct {
+	metav1.TypeMeta   `json:",inline" yaml:",inline"`
+	metav1.ObjectMeta `json:"metadata" yaml:"metadata"`
+
+	// Spec defines the desired state of ApisixPluginConfigSpec.
+	Spec   ApisixPluginConfigSpec `json:"spec" yaml:"spec"`
+	Status ApisixStatus           `json:"status,omitempty" yaml:"status,omitempty"`
+}
+
+// ApisixPluginConfigLabels was key/value pairs to specify attributes
+// and the maxProperties could only be 16.
+type ApisixPluginConfigLabels map[string]string
+
+// ApisixPluginConfigSpec defines the desired state of ApisixPluginConfigSpec.
+type ApisixPluginConfigSpec struct {
+	// Id must be required.
+	Id int32 `json:"id" yaml:"id"`
+	// Labels use ApisixPluginConfigLabels
+	Labels ApisixPluginConfigLabels `json:"labels,omitempty" yaml:"labels,omitempty"`
```

Comment:
🤔 What's the purpose of using labels here?

Reply:
Well, Labels do not have to be exposed to users, labels for APISIX objects are used for the management purpose.

Reply:
I copied labels from the [`https://github.com/apache/apisix/blob/master/apisix/schema_def.lua#L830`](https://github.com/apache/apisix/blob/master/apisix/schema_def.lua#L830) you had provided before. 
I thought what I should do was just to translate it from lua to go.
i will delete it.

Reply:
We may take over the labels for apisix objects, but the users of CRD(s) don't have this because Kubernetes already have the label mechanism.